README.md

```diff
@@ -1 +1,171 @@
-# JavaScript-capstone-project
\ No newline at end of file
+<a name="readme-top"></a>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 JavaScript-capstone-project <a name="about-project"></a>
+
+**JavaScript-capstone-project** In this activity I set up a JavaScript project for a our capstone project
+
+## 🛠 Built With <a name="built-with"></a>
+
+- HTML
+- CSS
+- Javascript
+- webpack
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Hit_the_Api]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://github.com/SulimanJoya/JavaScript-capstone-project.git)
```

Comment:
[OPTIONAL] Your live demo link is not working. Kindly make sure your app is hosted and link updated to make your project look more professional and up to standard. :+1: